### DISCUSSION TITLE
Getting only the functions called during Python scripts runtime

### DISCUSSION THREAD

**Discussion by johnammon75** [ionelmc/python-hunter#122] [Q&A]
Howdy! 

Is it possible to only keep track of the functions called and not include their arguments? As is when filtering by "kind = call" the output contains information about the function arguments on top of the function name. 

I am looking for functionality similar to that of cProfiler if that helps explain what I am wanting to accomplish.

**Comment by ionelmc** [comment]
You might want to just run in profile mode then (eg: `hunter.trace(profile=True)`).
